lib/generators/metka/strategies/materialized_view/materialized_view_generator.rb

```diff
@@ -32,12 +40,19 @@ def source_table_name
             options[:source_table_name]
           end
 
-          def source_column_name
-            options[:source_column_name]
+          def source_columns
+            options[:source_columns]
+          end
+
+          def source_columns_names
+            source_columns.join('_and_')
           end
 
           def view_name
-            "tagged_#{source_table_name}"
+            return options[:view_name] if options[:view_name]
+
+            columns_sequence = source_columns == DEFAULT_SOURCE_COLUMNS ? nil : "_with_#{source_columns_names}"
```

Comment:
I'm not 100% sure if that logic is even necessary, but right now it provides some kind of backward compatibility: if it's just a "default" view generated, for a single tagged column `tags` and without specifying the view name, then it would behave like it was before multicolumn aggregations support, generating just `tagged_#{table_name}`, but if one or more columns would be provided as `source_columns`, then it would be `tagged_with_#{column1}_and_#{column2}_#{table_name}`